docs-source/content/userguide/istio/istio.md

```diff
@@ -89,59 +82,101 @@ metadata:
     weblogic.domainUID: domain2
 spec:
   ... other content ...
-  experimental:
+  configuration:
     istio:
       enabled: true
       readinessPort: 8888
 ```
 
-To enable the experimental Istio support, you must include the `istio` section
+To enable the Istio support, you must include the `istio` section
 and you must set `enabled: true` as shown.  The `readniessPort` is optional
-and defaults to `8888` if not provided.  
+and defaults to `8888` if not provided, it is used for readiness health check.
 
 ##### How Istio-enabled domains differ from regular domains
 
-Istio enforces a number of requirements on pods.  When you enable Istio support, the
-domain on a persistent volume sample scripts will make the following adjustments
-to your domain in order to satisy Istio's requirements:
-
-* On the Administration Server:
-    * Create a channel called `istio-probe` with listen address `127.0.0.1:8888` (or
-      the port you specified in the `readinessPort` setting).
-    * Create a channel called `istio-t3` with listen address `127.0.0.1` and the port
-      you specified as the admin port.
-    * Create a channel called `istio-ldap` with listen address `127.0.0.1` and the port
-      you specified as the admin port, with only the LDAP protocol enabled.
-    * Create a channel called `istio-T3Channel` with listen
-      address `127.0.0.1` and the port you specified as the T3 port.
-* In the server template that is used to create Managed Servers in clusters:
-    * Create a channel called `istio-probe` with listen address `127.0.0.1:8888` (or
-      the port you specified in the `readinessPort` setting) and the public address
-      set to the Kubernetes Service for the Managed Server.
-    * Create a channel called `istio-t3` with listen address `127.0.0.1` and the port
-      you specified as the admin port and the public address
-      set to the Kubernetes Service for the Managed Server.
-    * Create a channel called `istio-cluster` with listen address `127.0.0.1` and the port
-      you specified as the admin port, with only the CLUSTER_BROADCAST protocol enabled,
-      and the public address set to the Kubernetes Service for the Managed Server.
-    * Create a channel called `istio-http` with listen address `127.0.0.1:31111` and the
-      public address set to the Kubernetes Service for the Managed Server. Note that `31111`
-      is the Istio proxy (envoy) port.
-* The create domain job will be configured to disable injection of the Istio sidecar.
-
-Additionally, when the experimental Istio support is enabled for a domain, the operator will
+Istio enforces a number of requirements on pods.  When you enable Istio support in the domain resource, the
+introspector job automatically creates configuration overrides with the necessary channels for the domain to satisfy Istio's requirements, including:
+
+When deploying a domain with Istio sidecare injection enabled.  WebLogic Operator automatically add the following network
```

Comment:
sidecare -> sidecar

Reply:
Either "The operator" or "The WebLogic Server Kubernetes Operator"

Reply:
fixed